data/json/recipes/recipe_food.json

```
@@ -4699,7 +4699,7 @@
     "skill_used": "cooking",
     "difficulty": 5,
     "time": "30 m",
-    "autolearn": true,
+    "book_learn": [ [ "fermenting_book", 4 ] ],
```

```suggestion
    "book_learn": [ [ "fermenting_book", 2 ], [ "cookbook_foodfashions", 4 ] ],
```
The principles of krauting aren't too hard, but it's easy to mess up.